scripts/fast-reboot

```diff
@@ -679,6 +679,14 @@ then
   systemctl stop "$service_name"
 fi
 
+if [[ "$REBOOT_TYPE" = "fast-reboot" ]]; then
+    # Remove FLEX_COUNTER_TABLE from config_db.json
+    # This is done so that in fast-reboot recovery path, FLEX_COUNTER polling is delayed.
+    # Delayed FLEX_COUNTER polling is an attempt keep dataplane downtime below 30s threshold
+    jq --indent 4  'del(.FLEX_COUNTER_TABLE)' ${CONFIG_DB_FILE} > ${CONFIG_DB_FILE}.new
```

Comment:
How do we restore the config, if use has made some configs?

Reply:
For branches (201811, 201911, 202012):
The user changed configuration to `FLEX_COUNTER_TABLE` will be reset. This is acceptable so far as for the upgrades, we anyway delete the `/host/old_config/config_db.json` file (basically removing all of the old config, and letting new image take the default config).

For newer branches (2021**):
Below PRs are targeted to address the issue of keeping the user configurations after reboot. The idea is to maintain a delay indicator that OA will check and decide to enable/disable flex counter polling.

Adding you to the discussion thread about this topic.